no-computer.md

```diff
@@ -1,69 +1,170 @@
 # No computer project
 
 
+
+Blueprints for a building. Pretty straightforward.
+
+
+
 ## The user and a language
+
 This section describes who the project would serve and why a language might be a
+
 good way to meet their needs.
 
 
+
+This would serve architectual engineers, construction workers, and other people who would need to know exactly what to do to to create the building.
+
+
+
 ### What's the need?
+
 _What need is met by your idea? Who are you helping? What is that person's
+
 experience like now? What would their experience be like if you could help 
+
 them?_
 
 
+
+Everyone listed above. Without blueprints, it may be possible to create buildings, but not skyscrapers. Anything that takes into account the laws of physics would probably need detailed planning.
+
+
+
 ### Why a language?
+
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
 
+
+It lets them order a more accurate amount of materials needed and plan out the order to build things in.
+
+
+
 ### Why you?
+
 _What excites you about this idea? How did you come up with it?_
 
 
+
+It seemed like the easiest thing to think of...
+
+
+
 ### Domain
+
 _Describe the project's domain in five words._
 
 
+
+Design and creation of buildings?
+
+
+
 ### Interface (syntax)
+
 _How might the user interact with the language? What does programming look 
+
 like? Why is this the right way to interact with the problem domain?_ 
 
 
+
+A "program" would be the finished blueprint, so to interact with the language would be to draw the building on grid paper with specifications of length and type of materials. 
+
+
+
 ### Operation (semantics)
+
 _What might happen when a program runs? How does a program interact with the
+
 user? What kinds of errors might occur, and how might they be communicated to
+
 the user?_
 
 
+
+When a program "runs," people execute the building plans, put together the pieces specified in the blueprint, and voila, a building is made. Some errors would be buying the wrong materal, or having the wrong size for any piece. The construction workers would figure this out while trying to use the material/piece. More abstractly, there may be an error in physics calculations, and the building would topple.
+
+
+
 ### Expressiveness
+
 _What should be easy to do in this language? What should be possible, but
+
 difficult? What should be impossible or very difficult?_
 
 
+
+It should be easy to design a building. It's technically possible to make blueprints of nearly anything, but this depends on how lenient the definition of "building" is and where the line between "building blueprints" and "general-purpose blueprints" are.
+
+It should be impossible to do... anything outside of designing/planning the creation of physical objects.
```

Comment:
Mm, algorithms are out then.

An interesting thought experiment is whether one can describe 'physical' objects that could not be constructed in the 3D Cartesian geometry we live in.

Can we draw blueprints or specifications for klein bottle? For Escher's steps?